### DISCUSSION TITLE
Per device settings for "Use system color scheme" & per device themes

### DISCUSSION THREAD

**Discussion by DarkArc** [standardnotes/forum#3145] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
Yes, my desktop is always in the dark theme. However, I can't read code in _any_ of the standard notes dark themes (due to my protonopia). So, for better readability, I like to use the light theme on my desktop.

However, on my phone I have totally different needs, namely, I don't want to be caught off guard opening the app in the middle of the night, and being blinded by a bright white theme.

**Describe the solution you'd like**
I'd like the "Use system color scheme" setting to be per-device, as different "systems" have different needs.

**Describe alternatives you've considered**
An alternative (that should happen anyways, is improving the dark themes for colorblind folks). _However_, even with that, I can understand wanting to keep standard notes themed a certain way on a certain device (different devices have different color schemes).

Overall, I was very surprised this setting synced between devices.

**Comment by ningsongshen** [comment]
Yes I'd agree! Automatic theme switching is a setting I prefer to be kept local on a device rather than synced.

I also sometimes face an issue that can be described as follows:
- My laptop switches themes from light to dark in the evening
- Standard Notes on desktop switches accordingly
- Then my phone (iOS) swiches from light to dark
- Opening Standard Notes on phone later at night also makes the switch
- But the toggle seems to untoggle the theme switch on desktop, leaving me with a bright white note app on desktop

Sometimes two of the 'switching in 5 seconds' windows pop up which undoes the mode switching.

I also believe theme switching should be automatic, and not take 5 seconds (but this is in another thread!)
